react/features/always-on-top/AlwaysOnTop.js

```diff
@@ -161,27 +173,25 @@ export default class AlwaysOnTop extends Component<*, State> {
      * @returns {ReactElement}
      */
     _renderVideoNotAvailableScreen() {
-        const { avatarURL, displayName, isVideoDisplayed } = this.state;
+        const { avatarURL, displayName, formattedDisplayName, isVideoDisplayed } = this.state;
 
         if (isVideoDisplayed) {
             return null;
         }
 
         return (
             <div id = 'videoNotAvailableScreen'>
-                {
-                    avatarURL
-                        ? <div id = 'avatarContainer'>
-                            <img
-                                id = 'avatar'
-                                src = { avatarURL } />
-                        </div>
-                        : null
-                }
+                <div id = 'avatarContainer'>
+                    <StatelessAvatar
```

Comment:
Do you need to calculate the avatar color here?

Reply:
Right, I do, thanks!